test/test_torch.py

```diff
@@ -14628,6 +14628,144 @@ def test_ones_like_multiple_device(self, devices):
         self.assertEqual(output, expected)
 
 
+# Tests that ops which should return views do return views
+class TestViewOps(TestCase):
+
+    def test_diagonal_view(self, device):
+        t = torch.ones((5, 5))
```

Comment:
Yep, just a mistake in this version of the PR. I'm going to fix it when I add a test or two for the bracket operator (to avoid hitting the CI again).